src/fmt.rs

```diff
@@ -0,0 +1,23 @@
+use termcolor::{Color, ColorChoice, ColorSpec, StandardStream, WriteColor};
+
+pub fn bold<F>(write: F)
+where
+    F: FnOnce(&mut StandardStream) -> std::io::Result<()>,
+{
+    let mut stdout = StandardStream::stdout(ColorChoice::Auto);
+    stdout.set_color(ColorSpec::new().set_bold(true)).ok();
+    write(&mut stdout).expect("Unexpected error writing to stdout");
+    stdout.reset().ok();
```

Comment:
A more conventional approach to ignoring `Result`s than `ok()` is to use `let _`, e.g.

```rust
let _ = stdout.set_color(ColorSpec::new().set_bold(true));
write(&mut stdout).expect("Unexpected error writing to stdout");
let _ = stdout.reset();
```

This avoids the conversion to `Option`, and I think is a generally renown approach to indicating that `Result`s are being ignored.

Also add a comment mentioning why we're ignoring these.